domains/runtime/evm/src/lib.rs

```diff
@@ -361,7 +383,7 @@ impl domain_pallet_executive::ExtrinsicStorageFees<Runtime> for ExtrinsicStorage
     }
 
     fn on_storage_fees_charged(charged_fees: Balance) {
-        OperatorRewards::note_operator_rewards(charged_fees)
+        BlockFees::note_domain_execution_fee(charged_fees)
```

Comment:
This is coming from `pallet-executive` for failed extrinsics at pre-dispatch and we charge the storage fees. This should be split between farmers and operators since the charged fees is taken from `FinalDomainTransactionByteFee` I believe.

Would also need to resolve a todo there

Reply:
We are charging the domain user a 3x consensus storage fee to cover the cost brought by duplicated/illegal tx thus I think it is okay to not charge the consensus storage fee again, WDYT?

Reply:
I'm not sure that would be good idea to remove this. While duplicated transactions occurs since operators might pick the same extrinsics, this variant of the transaction is completely controlled by attacker where after every bundle produced, attacker can submit trsaction with same nonce and higher amount and operators will end up picking it anyway.

thoughts @dariolina

Reply:
Honestly I'm not sure what "remove this" is discussed here.

Reply:
>Honestly I'm not sure what "remove this" is discussed here.

It is about whether we should charge the user for the consensus storage fee of illegal tx (same nonce with higher tip). Thinking more on this, it makes sense to me now, since we are not support the "same nonce with higher tip" tx it makes sense to charge the user for the full cost.